model/host/host.go

```diff
@@ -1179,12 +1179,9 @@ func GetNumContainersOnParents(d distro.Distro) ([]ContainersOnParents, error) {
 	// parents come in sorted order from soonest to latest expected finish time
 	for i := len(allParents) - 1; i >= 0; i-- {
 		parent := allParents[i]
-		currentContainers := []Host{}
-		if parent.Status == evergreen.HostRunning {
-			currentContainers, err = parent.GetContainers()
-			if err != nil {
-				return nil, errors.Wrapf(err, "Could not find containers for parent %s", parent.Id)
-			}
+		currentContainers, err := parent.GetContainers()
```

Comment:
We can talk about this in person if you want:

I previously changed the logic of this function to include parents that are not running (i.e. not initialized), because the point of creating new parents in `InsertParentIntentsAndGetNumHostsToSpawn` is so we can spawn our desired number of containers--otherwise we only use parents that existed before this and we don't actually spawn the number of hosts we would have wanted.  

If the container tries to start up before the parent, this job will retry.